go.mod

```diff
@@ -4,41 +4,41 @@ go 1.18
 
 require (
 	github.com/davecgh/go-spew v1.1.1
-	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.19.0
-	k8s.io/api v0.24.4
-	k8s.io/apimachinery v0.24.4
-	k8s.io/client-go v0.24.4
-	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9
+	github.com/onsi/ginkgo/v2 v2.2.0
+	github.com/onsi/gomega v1.20.1
+	k8s.io/api v0.25.0
+	k8s.io/apimachinery v0.25.0
+	k8s.io/client-go v0.25.0
+	k8s.io/utils v0.0.0-20220728103510-ee6ede2d64ed
```

Comment:
What caused these bumps? I'm happy to bump the k8s dependencies, but if we are we might as well bump `controller-runtime` to `v0.13.0`

Reply:
Oh interesting. I'll double check what happened.